src/RowCollection.php

```diff
@@ -270,7 +271,7 @@ public function offsetGet($offset)
      */
     public function rewind()
     {
-        return reset($this->rows);
+        reset($this->rows);
```

Comment:
This method must return the value of reset

Reply:
PHP docs say otherwise. Please see https://www.php.net/manual/en/iterator.rewind.php